specification/graph/resource-manager/Microsoft.Graph/preview/1.0/guest.json

```diff
@@ -0,0 +1,590 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GraphManagementClient",
+    "description": "The Microsoft Graph API provides a RESTful set of web services that interact with Microsoft Graph service to manage your entities and relationships across different services exposed with rich scenarios around users and groups (Azure Active Directory).",
+    "version": "1.0",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true
+    }
+  },
+  "host": "graph.microsoft.com",
+  "basePath": "/v1.0",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/invitations": {
+      "post": {
+        "tags": [
+          "GuestUser"
+        ],
+        "operationId": "GuestUser_Create",
+        "description": "Create a GuestUser sending an invitation",
+        "parameters": [
+          {
+            "name": "invitationContent",
+            "in": "body",
+            "required": true,
+            "description": "The email address of the GuestUser you want to invite, the domain of the mail address must exist on existing Azure AD",
+            "schema": {
+              "$ref": "#/definitions/invitationSended"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Create successful. The operation returns the resulting GuestUser resource.",
+            "schema": {
+              "$ref": "#/definitions/invitation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./graph.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Guest User": {
+            "$ref": "./examples/GuestUserCreate.json"
+          }
+        }
+      }
+    },
+    "/users/{guestUserId}": {
```

Comment:
these seem like dataplane APIs. please make sure the API paths are ARM compliant if this indeed is a resource provider and has control plane APIs. best to schedule a call to go over this.

Reply:
Where can I call them ? because I'll be not available from 21 august to 22 september, so if its not today or tomorrow it will be next month :/

Reply:
I think I will stop here bacause terraform azuread will change their mind and use API directly without using azure-sdk-for-go : https://github.com/terraform-providers/terraform-provider-azuread/pull/297#issuecomment-676852248 

So I'm not sure my work will be used except for us, I already use my generated terraform provider based on this PR and the generated go code, untill the publish their version with new strategy of using API directly will published.